scipy/stats/_multivariate.py

```diff
@@ -1717,6 +1719,12 @@ class wishart_gen(multi_rv_generic):
     distribution :math:`W_1(\nu, 1)` collapses to the :math:`\chi^2(\nu)`
     distribution.
 
+    The algorithm [2]_ implemented by the `rvs` method occasionally
```

Comment:
As long as this is consistent with what is available in MATLAB and R I agree we should extend the range of `df`.

I would hesitate to say "occasionally", as singular matrices are produced nearly every time I run this with `p - 1 < \nu < p` -- and those that are not singular are functionally singular (extremely ill-conditioned).  It might be more useful to say that singular matrices are produced under this condition with occasional non-singular (but ill-conditioned) matrices occurring due to floating point error.

After looking around a bit, I find that the Wishart distribution is a distribution of positive semidefinite random matrices (as [defined here](https://www.math.wustl.edu/~sawyer/hmhandouts/Wishart.pdf) at least), which means that their eigenvalues can be zero making them non-invertible.  I believe `p - 1 < \nu < p` is the mode where these non-invertible matrices arise.  This does not appear to be the fault of the algorithm

Reply:
> After looking around a bit, I find that the Wishart distribution is a distribution of positive semidefinite random matrices

Yeah I was finding conflicting information about that. If we can confirm that this is correct behavior and not a shortcoming of the algorithm, then we don't have to mention anything in the notes about it producing singular matrices, as that is the expected result. We would actually just change the parts of the documentation that state it is a distribution over of positive _definite_ matrices to positive _semidefinite_ matrices, right?

I held back from that because on Wikipedia:
![image](https://user-images.githubusercontent.com/6570539/104965389-7e5caf00-5993-11eb-9b30-63b854aaafbe.png)

but of course, it might not be correct in specifying that the support is only the positive definite matrices. After all, it also says:

> It is a family of probability distributions defined over symmetric, nonnegative-definite matrix-valued random variables (“random matrices”).

which sounds more like positive semidefinite than positive definite.

Reply:
@ChadFulton do you happen to know the answer here? We're wondering if the support is actually the positive semidefinite matrices when `p-1 < n < p`.